azure/public_ip.sh

```diff
@@ -0,0 +1,24 @@
+# Public IP related functions
+
+fn azure_public_ip_create(name, group, location, allocation) {
+       (
+               az network public-ip create --name $name --resource-group $group --location $location --allocation-method $allocation
```

Comment:
Another thing: as you said in a previous PR, nash will have a way to get `stderr` in a variable, but right now I would have to get it like your proposal (`out <= cmd >[2=1]`). Don't you think we should wait and do this with `stderr` in a variable for everybody?